<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C02QG38VG8WP" ignored="0" name="APITest" tests="11" failures="2" timestamp="25 Jan 2019 15:00:47 GMT" time="11.349" errors="0">
  <testcase name="createUser" time="0.93" classname="com.apitest.Segment.UserTest"/>
  <testcase name="createUser" time="0.895" classname="com.apitest.Segment.UserTest"/>
  <testcase name="createUser" time="0.939" classname="com.apitest.Segment.UserTest"/>
  <testcase name="createUser" time="0.882" classname="com.apitest.Segment.UserTest"/>
  <testcase name="createUser" time="0.894" classname="com.apitest.Segment.UserTest"/>
  <testcase name="getAllUser" time="0.512" classname="com.apitest.Segment.UserTest"/>
  <testcase name="getUserDetail" time="0.9" classname="com.apitest.Segment.UserTest"/>
  <testcase name="getUserDetail" time="0.895" classname="com.apitest.Segment.UserTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.apitest.Segment.UserTest.getUserDetail(UserTest.java:65)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getUserDetail -->
  <testcase name="getUserDetail" time="0.898" classname="com.apitest.Segment.UserTest"/>
  <testcase name="getUserDetail" time="0.515" classname="com.apitest.Segment.UserTest"/>
  <testcase name="getUserDetail" time="0.521" classname="com.apitest.Segment.UserTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.apitest.Segment.UserTest.getUserDetail(UserTest.java:65)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getUserDetail -->
</testsuite> <!-- APITest -->
